cypher/cypher-docs/src/docs/dev/ql/administration/security/deny-match-syntax.asciidoc

```diff
@@ -2,12 +2,12 @@
 [source, cypher]
 -----
 DENY MATCH
-    "{" { * | properties } "}"
+    "{" { * | property[, ...] } "}"
```

Comment:
Just out of curiosity, why is the syntax here [, ...], when it's [,...] (without a space separating the comma and the periods) on the other lines in the example? 
This applies to all of the `[, ...]` in this PR.

Reply:
I know that I in some of my PRs (not sure if this one or another) went over to try and make sure we had `thing[, ...]` instead of `thing [, ...]` but I guess I missed going over and checking `[, ...]` vs `[,...]` (and therefore probably also `thing [,...]`).

I know I prefer having the space`[, ...]` but I guess different people have added it differently which is why we now have a mix.

I guess I could (add a card to) go over and make sure its consistent

Reply:
Since it looks like its on these PRs I might just go over and fix it here

Reply:
Added an additional commit which updates so we should have `[, ...]` everywhere now :)